test/rdfTypes/VariableTest.cpp

```diff
@@ -76,20 +76,20 @@ TEST(Variable, ScoreAndMatchVariablesUnicode) {
 // _____________________________________________________________________________
 TEST(Variable, ScoreAndMatchUnicodeExhaustive) {
 #ifndef QLEVER_RUN_EXPENSIVE_TESTS
-  GTEST_SKIP();
+  // GTEST_SKIP();
```

Comment:
This doesn't seem correct. Either you remove the whole preprocessor thingy and run them always, or you uncomment this line